firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageViewController.swift

```diff
@@ -193,22 +208,52 @@ final class NativeErrorPageViewController: UIViewController,
             ),
             logoImage.widthAnchor.constraint(equalToConstant: UX.logoSizeWidth)
         ]
-        NSLayoutConstraint.activate(contraintsList)
+
+        iPadContraintsList = [
+            scrollContainer.topAnchor.constraint(
+                equalTo: scrollView.topAnchor,
+                constant: UX.iPadPadding.top
+            ),
+            scrollContainer.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+                constant: UX.iPadPadding.leading
+            ),
+            scrollContainer.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                constant: UX.iPadPadding.trailing
+            ),
+            scrollContainer.bottomAnchor.constraint(
+                equalTo: scrollView.bottomAnchor,
+                constant: UX.iPadPadding.bottom
+            ),
+            logoImage.widthAnchor.constraint(equalToConstant: UX.logoSizeWidthiPad),
+            reloadButton.widthAnchor.constraint(
+                equalTo: commonContainer.widthAnchor,
+                multiplier: UX.reloadButtonIpadMultiplier
+            )
+        ]
+
+        NSLayoutConstraint.activate(commonContraintsList)
+
+        if UIDevice().userInterfaceIdiom == .pad {
```

Comment:
Shouldn't we be checking the trait collection (for `compact` vs `regular`) rather than the user interface idiom? Just wondering how the UI should render in the case of iPad windows that are sized very small, for example (in that case I'd expect we probably want to use the iPhone constraints).

Reply:
Thank you for pointing it out! I will make the necessary changes.